ext/intl/php_intl.stub.php

```diff
@@ -349,11 +349,8 @@ function normalizer_get_raw_decomposition(string $string, int $form = Normalizer
 
 function resourcebundle_create(?string $locale, ?string $bundle, bool $fallback = true): ?ResourceBundle {}
 
-/**
- * @param string|int $index
- * @return mixed
- */
-function resourcebundle_get(ResourceBundle $bundle, $index, bool $fallback = true) {}
+/** @param string|int $index */
+function resourcebundle_get(ResourceBundle $bundle, $index, bool $fallback = true): mixed {}
```

Comment:
As far as I saw, only a `float` return type is impossible, so using `mixed` seems to be warranted.